melkor/models/sklearn_pipeline.py

```diff
@@ -0,0 +1,110 @@
+import sklearn
+import pandas as pd
+import numpy as np
+import pickle
+from nptyping import NDArray, Float32
+from typing import Union
+from sklearn.pipeline import Pipeline
+from sklearn.compose import ColumnTransformer
+
+# TODO: Add scoring metrics
+# TODO: Change y to 1d array
+
+
+class SKLearnModelPipeline:
+    def __init__(
+        self,
+        config: dict = {},
+        cat_cols: list = [],
+        num_cols: list = [],
+        inference: bool = False,
+    ) -> None:
```

Comment:
I get the befits of the soft typing for the IDE and linters, etc but when there is no return its just unnecessary (only personal opinion)